user-guide/settings/privacy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Privacy
+slug: privacy-submenu
+tags: privacy, delete account, unsubscribe, email list
+publish: true
+position: 40
+---
+
+# Privacy submenu
+
+The __Privacy__ submenu provide options to opt-out of receiving emails from Progress and its partners and the option to delete your Fiddler Everywhere associated account.
```

Comment:
1) provide**s** with "s" (or alternatively, use **contains**)
2) Rephrase "... the option to delete your ..." to avoid repeating "option" and also specifying that they can **request the deletion**, not delete it themselves (at least that's how I would expect it to work). For example:
... and the opportunity to request the deletion of your Fiddler ...